### PR TITLE
WEBUI-724: add pr checks for 10.10

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: .github/workflows/ftest.yaml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    branches:
+      - '10.10'
+  pull_request:
+    branches:
+      - '10.10'
+
+env:
+  REFERENCE_BRANCH: '10.10'
+  ELEMENTS_REFERENCE_BRANCH: 'maintenance-2.4.x'
+  NPM_REPOSITORY: https://packages.nuxeo.com/repository/npm-public/
```
Comment (semisse) [inline .github/workflows/ftest.yaml]: don't we need to pass this value as a string?

File: .github/workflows/ftest.yaml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+          bower install && npm install
+          popd
+
+      - name: Lint Web UI
+        run: |
+          pushd nuxeo-web-ui
+          npm run lint
+          popd
+
+      - name: Link elements to Web UI
```
Comment (semisse) [inline .github/workflows/ftest.yaml]:
```suggestion
      - name: Link Nuxeo Elements to Web UI
```

**Review by semisse — DISMISSED** [review]

File: .github/workflows/ftest.yaml (hunk 1 above)
Comment (richardsd) [reply .github/workflows/ftest.yaml]: No, everything is a string, we should probably try to remove all the '' in here.

File: .github/workflows/ftest.yaml (hunk 2 above)
Comment (richardsd) [reply .github/workflows/ftest.yaml]: I didn't use `Nuxeo Elements` just to make it clear that we are linking all the core, ui and dataviz elements. 
Do you think that using `Nuxeo Elements` is clear enough? 
Thanks Sam

File: .github/workflows/ftest.yaml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+      - name: Archive packages
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: packages
+          path: |
+            # HACK - removed the SNAPSHOT because of sed
+            # plugin-nuxeo-web-ui/marketplace/target/nuxeo-web-ui-marketplace-*-SNAPSHOT.zip
+            plugin-nuxeo-web-ui/marketplace/target/nuxeo-web-ui-marketplace-*.zip
+            plugin-nuxeo-web-ui/marketplace-itests/target/nuxeo-web-ui-marketplace-itests-*.zip
```
Comment (richardsd) [inline .github/workflows/ftest.yaml]: Missing a new line here.

**Review by semisse — APPROVED** [review]

File: .github/workflows/ftest.yaml [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+                <server>
+                  <id>maven-internal</id>
+                  <username>${{ secrets.PACKAGES_AUTH_USER }}</username>
+                  <password>${{ secrets.PACKAGES_AUTH_TOKEN }}</password>
+                </server>
+              </servers>
+              </settings>' > ~/.m2/settings.xml
+
+      - name: Checkout the nuxeo-elements repo
+        uses: actions/checkout@v2
```
Comment (Gabez0r) [inline .github/workflows/ftest.yaml:121]: Any particular reason not to be using v3? I'm ok with using v2 for consistency with other, btw.

**Review by Gabez0r — APPROVED** [review]
LGTM

File: .github/workflows/ftest.yaml (hunk 4 above)
Comment (richardsd) [reply .github/workflows/ftest.yaml:121]: No particular reason. It was released last month and the major difference is the nodejs runtime update to version 16 https://github.com/actions/checkout#whats-new. I'm ok with using v2 for consistency too.

**Pull request merged**